packages/react/src/time-input/docs/overview.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Time input
+description: The Time input component allows users to enter a time free-form, and in various formats.
+group: Forms
+storybookPath: /story/Forms-TimeInput--basic
+unreleased: true
+figmaGalleryNodeId: REPLACE_ME
+relatedComponents: ['text-input']
+---
+
+```jsx live
+() => {
+	const [value, setValue] = React.useState({ raw: '' });
+
+	const onChange = (timeValue) => {
```

Comment:
I disagree with this. We shouldn't be promoting `useEffect` like this imo. Devs should know that `onChange` accepts a function, and that function can be anything.

Doing this would also make the invalid one more complicated, because now that would have to managed in state and set in the `useEffect`